packages/amplify-cli/scripts/post-install.js

```diff
@@ -18,11 +18,11 @@ console.log(chalk.green('Successfully installed the Amplify CLI'));
 console.log(chalk.green('----------------------------------------'));
 console.log('\n');
 
-console.log(chalk.green('JavaScript Getting Started - https://aws-amplify.github.io/docs/js/start'));
+console.log(chalk.green('JavaScript Getting Started - https://docs.amplify.aws/start'));
```

Comment:
Getting started for JS is here: ```https://docs.amplify.aws/start/q/integration/js```, perhaps the text should be changed from "JavaScript" to "Frontend" ?

Reply:
I think this is fine. This will be more explicit. So resolving as "by design" for now.